Packetery/Checkout/Model/Carrier/Facade.php

```diff
@@ -143,7 +183,7 @@ private function getDynamicCarrier(AbstractCarrier $carrier, ?int $carrierId): ?
      * @param string $carrierCode
      * @return \Packetery\Checkout\Model\Carrier\AbstractCarrier
      */
-    private function getMagentoCarrier(string $carrierCode): AbstractCarrier {
+    public function getMagentoCarrier(string $carrierCode): AbstractCarrier {
```

Comment:
Vypadá to, že může vracet i bool.

Reply:
Nemůže. Pokud Magento API vrátí bool, tak chci aby to umřelo na TypeErroru.

Reply:
A nebylo by lepší to přiznat výjimkou?

Reply:
Přijde mi to zbytečné. Z call stacku a typu poznáš o co jde. Takto tam je méně řádek.